wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationTableViewDataSource.swift

```diff
@@ -514,4 +521,14 @@ extension ConversationTableViewDataSource {
         return !Calendar.current.isDate(current, inSameDayAs: previous)
     }
 
+    private func IsTimeIntervalDifferentFromPreviousToCurrentMessage(for message: ZMConversationMessage, at index: Int) -> Bool {
```

Comment:
I suggest to express this as an extension of `Date`: 

```swift
extension Date {

  func isInSameMinute(asDate date: Date) -> Bool {
    let calendar = Calendar.current
    let components = calendar.dateComponents([.year, .month, .day, .hour, .minute], from: self)
    let otherComponents = calendar.dateComponents([.year, .month, .day, .hour, .minute], from: date)
    return components == otherComponents
  }

}
```

Then you can call it like:

```swift
let isInSameMinute = message.serverTimestamp.isInSameMinute(asDate: previousMessage.serverTimestamp)
```

Reply:
Done!